docs/source/install-linux.rst

```diff
@@ -0,0 +1,85 @@
+.. _install-linux:
+
+####################
+Installation - Linux
+####################
+
+.. important:: THIS IS THE COZMO SDK BETA. The SDK is under development and is subject to change.
+
+This guide provides instructions on installing the Cozmo SDK for computers running with an Ubuntu Linux operating system.
+
+.. warning:: The Cozmo SDK is tested and and supported on Ubuntu 14.04 and 16.04. Anki makes no guarantee the Cozmo SDK will work on other versions of Linux.  If you wish to try the Cozmo SDK on versions of Linux *other than* Ubuntu 14.04 or 16.04, please ensure the following dependencies are installed:
+
+  * Python 3.5.1 or later
+  * pip for Python 3 (Python package installer)
+  * Android command-line tools (https://developer.android.com/studio/index.html#Other)
+  * usbmuxd for iOS / :ref:`adb` for Android
+
+-------------------
+Python Installation
+-------------------
+
+^^^^^^^^^^^^
+Ubuntu 14.04
+^^^^^^^^^^^^
+
+1. Type the following into your Terminal window to install Python 3.5::
+
+    sudo add-apt-repository ppa:fkrull/deadsnakes
+    sudo apt-get update
+    sudo apt-get install python3.5
+    sudo update-alternatives --install /usr/bin/python3 python3.5 /usr/bin/python3.5.1
+
+2. Then install pip by typing in the following into the Terminal window::
+
+    sudo apt-get install python3-setuptools
+    sudo easy_install3 pip
+
+3. Last, install Tkinter::
+
+    sudo apt-get install python3.5-tk
+
+^^^^^^^^^^^^
+Ubuntu 16.04
+^^^^^^^^^^^^
+
+1. Type the following into your Terminal window to install Python::
+
+    sudo apt-get update
+    sudo apt-get install python3
+
+2. Then install pip by typing in the following into the Terminal window::
+
+    sudo apt install python3-pip
+
+3. Last, install Tkinter::
+
+    sudo apt-get install python3-pil.imagetk
+
+-------------------
+Mobile Device Setup
+-------------------
+
+4. **iOS** devices require `usbmuxd <https://github.com/libimobiledevice/usbmuxd>`_ in order to run the Cozmo SDK. Usbmuxd is required for the computer to communicate with the iOS device over a USB cable.
+
+5. **Android** devices require installation of :ref:`adb` (adb) in order to run the Cozmo SDK. Android Debug Bridge is required for the computer to communicate with the Android mobile device over a USB cable. This service automatically runs when required.
+
+----------------
+SDK Installation
+----------------
+
+To install the SDK, type the following into the Terminal window::
+
+    pip3 install --user cozmo[camera]
+
+Note that the [camera] option adds support for processing images from Cozmo's camera.
+
+^^^^^^^^^^^^^^^
+Troubleshooting
+^^^^^^^^^^^^^^^
+
+Please see the main :ref:`initial` page for tips, or visit the `Cozmo SDK Forums <https://forums.anki.com/>`_ to ask questions, find solutions, or for general discussion.
```

Comment:
Could these link to the #troubleshooting anchor in initial instead - e.g. instead of :ref:`initial` do :ref:`initial#troubleshooting` (i'm not sure what the Sphinx syntax is so that might not be how, but hopefully there is a way of doing this)